src/core/gui/MainWindow.cpp

```diff
@@ -349,10 +348,8 @@ void MainWindow::dragDataRecived(GtkWidget* widget, GdkDragContext* dragContext,
                     continue;
                 }
 
-                if (pixbuf) {
-                    win->control->clipboardPasteImage(pixbuf);
-
-                    g_object_unref(pixbuf);
+                if (pixbuf.get()) {
```

Comment:
CLibrariesSPtr (of which GObjectSPtr is a specialization) is convertible to bool. There should be no need for the `.get()` in cases like this one.
```suggestion
                if (pixbuf) {
```